source/server/listener_impl.cc

```diff
@@ -675,20 +675,6 @@ void ListenerImpl::buildFilterChains() {
 void ListenerImpl::buildConnectionBalancer(const Network::Address::Instance& address) {
   auto iter = connection_balancers_.find(address.asString());
   if (iter == connection_balancers_.end() && socket_type_ == Network::Socket::Type::Stream) {
-#ifdef WIN32
-    // On Windows we use the exact connection balancer to dispatch connections
-    // from worker 1 to all workers. This is a perf hit but it is the only way
-    // to make all the workers do work.
-    // TODO(davinci26): We can be faster here if we create a balancer implementation
-    // that dispatches the connection to a random thread.
-    ENVOY_LOG(warn,
-              "ExactBalance was forced enabled for TCP listener '{}' because "
-              "Envoy is running on Windows."
-              "ExactBalance is used to load balance connections between workers on Windows.",
-              config_.name());
-    connection_balancers_.emplace(address.asString(),
-                                  std::make_shared<Network::ExactConnectionBalancerImpl>());
-#else
```

Comment:
prior for windows the code below never ran, will we run into issues now in the new version since it's not defined out?

Reply:
The old logic:
- check WIN or not
- force use Exact connection balancer if WIN
- use one balancer refer to config if not WIN

The new logic:
- check config existed or not
- use one balancer refer to config if existed
- force use Exact connection balancer if WIN and config not existed

Reply:
so we now can incorrectly configure a connection balancer that doesn't work on WIN now, compared to before?